e2e_tests/tests/experiment/test_pytorch.py

```diff
@@ -1,64 +0,0 @@
-from typing import List
-
-import pytest
-
-from tests import api_utils
-from tests import config as conf
-from tests import experiment as exp
-
-
-@pytest.mark.parallel
-@pytest.mark.e2e_slurm_gpu
-def test_pytorch_gradient_aggregation() -> None:
```

Comment:
This test was written before we had parallel gpu tests.  We didn't have a way to validate our gradient aggregation logic (which is ours, not just a library we call), other than an e2e test.

It never needed a slurm test.

This test has been obsoleted by [multi-gpu unit testing](https://github.com/determined-ai/determined/blob/f5400eadc8109ad5ac5885b50475c3df22676211/harness/tests/experiment/pytorch/test_pytorch_trial.py#L980-L982).